Sources/Components/ARCProgressBar.swift

```diff
@@ -81,15 +81,30 @@ struct CapsuleView: View {
         return 1 / (1 - (1 - leftOver))
     }
 
+    /// Calculate the top capsule width
+    var progressWidth: CGFloat {
+        let progress = isPartiallyFilled ? capsuleWidth / CGFloat(partiallyDividedBy) : capsuleWidth
+        return progress + (.ArcProgressBar.capsuleTrailingWidth / 2) // For perfect centering
+    }
+
     var body: some View {
         ZStack(alignment: .leading) {
             Capsule().fill(emptyColor)
             if isFilledCapsule {
-                Capsule().fill(filledColor)
-                    .frame(
-                        width: isPartiallyFilled ? capsuleWidth / CGFloat(partiallyDividedBy) : capsuleWidth,
-                        height: capsuleHeight
-                    )
+                ZStack {
+                    Capsule().fill(filledColor)
```

Comment:
Small, fine to leave. Would be nice is formatting was the same across all the views. Can be sorted later